### DISCUSSION TITLE
Random song/patch name generator

### DISCUSSION THREAD

**Discussion by withak** [SynthstromAudible/DelugeFirmware#2107] [Ideas]
I'm really bad at naming things, so it would be neat to have an auto-name option like Polyend does.  It seems like it just combines an adjective or adverb with a noun, and occasionally hyphenates them.  Not sure if there is more to it than that, or if the list of word options that it pulls from is curated in some way that makes it work better.

**Comment by m-m-adams** [comment]
I like this idea a lot!

**Comment by stephencarr** [comment]
Elektron just added this to their latest Syntakt firmware.  This would save a bunch of time for me. I don't really care to name work in progress, or new ideas, so this would serve well. 

Instead of the default being "New Song", it could just randomize a couple of words to make a name. 

Software releases are sometimes like this.
